api/lib/opentelemetry/context.rb

```diff
@@ -75,31 +96,26 @@ def with_value(key, value)
       #   to the block
       def with_values(values)
         ctx = current.set_values(values)
-        prev = ctx.attach
+        prev = attach(ctx)
         yield ctx, values
       ensure
-        ctx.detach(prev)
-      end
-
-      # Returns the value associated with key in the current context
-      #
-      # @param [String] key The lookup key
-      def value(key)
```

Comment:
Why lose this? This is just a convenience, right? I.e. instead of:
```ruby
OpenTelemetry::Context.current.value('foo')
```
we can
```ruby
OpenTelemetry::Context.value('foo')
```

Reply:
Unintentional, I'll restore it and add some test.